modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -10,6 +10,8 @@ any-category=Any Category
 any-folder=Any Folder
 any-site=Any Site
 any-user=Any User
+asset-tag-search-facet-display-context=Tag Facet Display Context
```

Comment:
Shouldn't the language key and the value match?

Reply:
You're right! I'll update this property.